tests/metagen/typegraphs/identities/py/fdk.py

```
@@ -0,0 +1,1920 @@
+        doc, variables, files = self.build_gql(mutation, "mutation", name)
+        result = self.fetch(doc, variables, opts, files)
+        return result["value"] if isinstance(inp, MutationNode) else result
+
+
+class GraphQLTransportUrlib(GraphQLTransportBase):
+    def fetch(
+        self,
+        doc: str,
+        variables: typing.Dict[str, typing.Any],
+        opts: typing.Optional[GraphQLTransportOptions],
+        files: typing.Dict[str, File] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)